clang/runtime/dpct-rt/include/dpct/dpl_extras/dpcpp_extensions.h

```diff
@@ -566,6 +566,58 @@ class radix_sort {
   uint8_t *_local_memory;
 };
 
+/// Load linear segment items into block format across threads
+/// Helper for Block Load
+enum load_algorithm{
+
+  subgroup_load,
+  workgroup_load
+
+};
+
+template <size_t GROUP_WORK_ITEMS,
+          size_t ITEMS_PER_WORK_ITEM,
+          load_algorithm ALGORITHM,
+          typename InputT,
+          typename InputIteratorT>
+class load {
+public:
+  template <typename Item>
```

Comment:
`Item` is unused here. This template argument should be removed.

Reply:
Removed where it is not needed.